tools/install-powershell.readme.md

```diff
@@ -0,0 +1,56 @@
+
+## Features of install-powershell.sh
```

Comment:
There are syntax errors in the markdown.  Can you please enable these files in [this test](https://github.com/PowerShell/PowerShell/blob/master/test/common/markdown/markdown.tests.ps1#L75) and fix the syntax errors)